extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/ola-bridge.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- OLA Bridge -->
+    <bridge-type id="ola-bridge">
+        <label>OLA Bridge</label>
+        <description>The Ola bridge represents a single DMX universe connected via OLA</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+   
+        <config-description>
+            <parameter name="address" type="text">
+                <context>network_address</context>
+                <label>Network Address</label>
+                <description>Network address of bridge, format: address[:port]. Default port is 9010.</description>
+                <required>false</required>
```

Comment:
No, this defaults to localhost. In 99% of the configurations of OLA this will be on the same machine.